.github/ISSUE_TEMPLATE/qa-for-exercise-modules.md

```diff
@@ -28,14 +28,15 @@ Current Version of Module (use the latest commit value):  {click on the PR and g
   * [ ] author name
   * [ ] email
   * [ ] module version number of at least 1.0.0 if first public version or if this is an update then an [appropriately incremented version number](versioning_guidelines.md)
-  * [ ] module_template_version number is up to date with the [current sample module](https://raw.githubusercontent.com/arcus/education_modules/main/a_sample_module_template/a_sample_module_template.md) -- if not, the module should be brought in line with any changes that have occurred to the module template before continuing with QA
+  * [ ] current_version_description is a short, present-tense text fragment without bullets, and gives a learner-centric quick overview of version changes from the learner perspective
+  * [ ] docs_version number replaces the former module_template_version number, and it is up to date with the current [docs.md](https://raw.githubusercontent.com/arcus/education_modules/main/docs.md) -- if not, the module should be brought in line with any changes that have occurred to the docs.md standard before continuing with QA
   * [ ] language
   * [ ] narrator
   * [ ] comment appropriately filled out
   * [ ] long_description appropriately filled out
   * [ ] learning_objectives appropriately filled out
-  * [ ] link to CSS (currently https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css)
-  * [ ] script to Font Awesome kit (currently https://kit.fontawesome.com/83b2343bd4.js)
+  * [ ] version_history contains 1-3 past versions with bulleted links to commit-specific documents, unless this is an initial version of a module, in which case version_history can be empty or absent.
```

Comment:
```suggestion
  * [ ] version_history contains 1-3 past versions with bulleted links to commit-specific documents, unless this is an initial version of a module, in which case version_history can be empty.
```

Reply:
For the GH Actions thing that will check for the presence of "required" front matter fields, it will work better to have version_history in there but empty rather than left off altogether